tests/test_patch_extraction.py

```diff
@@ -226,3 +227,112 @@ def test_points_patch_extractor_jp2(_sample_jp2):
     )
 
     assert np.all(data == saved_data)
+
+
+def test_fixed_window_patch_extractor():
+    """Test FixedWindowPatchExtractor for VF."""
+    file_parent_dir = pathlib.Path(__file__).parent
+    input_img = pathlib.Path(
+        file_parent_dir.joinpath("data/TCGA-HE-7130-01Z-00-DX1.png")
+    )
+
+    stride = (20, 20)
+    patch_size = (200, 200)
+    img = misc.imread(input_img)
```

Comment:
Do we need to test for not uniform stride and patch size?

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/97/commits/d52f278d91a027a668335ab74b1a443bd76185ec